exchange/seat_non_bids_test.go

```diff
@@ -0,0 +1,109 @@
+package exchange
+
+import (
+	"testing"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/exchange/entities"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSeatNonBidsAdd(t *testing.T) {
```

Comment:
Repeating a comment left by @SyntaxNode, but could you update the descriptions of these tests to not include spaces? (e.g. `nil seatNonBidsMap` --> `nil-seatNonBidsMap`)

Reply:
Done